clang/test/CIR/CodeGen/const-array.c

```diff
@@ -4,7 +4,7 @@ void bar() {
   const int arr[1] = {1};
 }
 
-// CHECK: cir.global "private" constant internal @bar.arr = #cir.const_array<[#cir.int<1> : !s32i]> : !cir.array<!s32i x 1> {alignment = 4 : i64}
+// CHECK: cir.global "private" constant internal @bar.arr = #cir.const_array<[#cir.int<1> : !s32i]> : !cir.array<!s32i x 1> {alignment = 4 : i64, dsolocal}
```

Comment:
My expectation is that dsolocal should be in the same tier as `contant`, `internal`, etc - alignment is the odd one here we never took the time to fix.

In general, if you are adding attributes, make sure you take some time to figure out how it plays with the others or ask me about it. Just adding it adhoc anywhere feels a bit careless (you had choices of how to print this and took the most convenient) and adds more work to my review (this comment could have been a LGTM).